corehq/form_processor/backends/sql/processor.py

```diff
@@ -64,12 +65,18 @@ def save_processed_models(cls, processed_forms, cases=None, stock_updates=None):
                 FormAccessorSQL.save_deprecated_form(processed_forms.deprecated)
 
             FormAccessorSQL.save_new_form(processed_forms.submitted)
-
             if cases:
                 for case in cases:
                     CaseAccessorSQL.save_case(case)
             for stock_update in stock_updates or []:
                 stock_update.commit()
+            cls._publish_changes(processed_forms, cases)
```

Comment:
not totally sure where a lot of these calls and functions belong. trying to isolate as much as possible to single calls so that we can move them around as we figure out where we think they belong.